tutorial.md

```diff
@@ -158,33 +162,40 @@ When closing an existing ticket, LGTM will send a request of the form...
     "transition": "close"
 }
 ```
-When reopening a ticket, the request will be of the form...
-```json
-{
-	"issue-id": external_issue_id,
-    "transition": "reopen"
-}
-```
-The Github API expects state to be specified as either `open` or `close`, so we first make sure that our terminology matches that expected by Github, and then send a simple `PATCH` request to the appropriate resource endpoint.
+This can be handled by sending a `PATCH` request to the existing Github issue.
 ```python
 if transition == 'create':
    ########
-else:
-    issue_id = json_dict.get('issue-id')
+if transition == 'close':
 
-    if transition == 'reopen':
-        transition = 'open'
+    issue_id = json_dict.get('issue-id')
 
     r = requests.patch(URL + '/' + issue_id,
                         json={"state": transition},
                         headers=HEADERS)
 
     return jsonify({'issue-id': issue_id}), r.status_code
 ```
+When reopening a ticket, the request will be of the same form, except with the transition `reopen`. This can be handled similary, with Github expecting the state to be given as `open`.
```

Comment:
Maybe change "This can be handled similary, with Github"
to: "This can be handled in a similar way to closing a ticket, but with Github